tutorials/plot_sleep.py

```diff
@@ -0,0 +1,281 @@
+"""
+Example on sleep data
+=====================
+
+.. note:: This code is taken from the analysis code used in [3]_. If you reuse
+          this code please consider citing this work.
+
+This tutorial explains how to perform a toy polysomnography analysis that
+answers the following question:
+
+    Given two subjects from the Sleep Physionet dataset [1]_ [2]_, namely
```

Comment:
The indent does not render that great in the HTML.

Reply:
I was trying to find a way to highlight the question, and that's the best solution I could come up with.